cagov/covid19
# Display errors for invalid search on What's Open widget

Hello! It’s very exciting that this site is open source. I wanted to try my hand at contributing to it. If my pull request is more trouble than it’s worth, please let me know. I know your team is very busy and I don’t want to waste your time.

## What’s changed
The What's Open widget now displays errors for invalid searches. I’ve implemented the behavior recommended in [this issue](https://github.com/cagov/covid19/issues/1839).

![Screen Shot 2020-12-09 at 11 10 28 AM](https://user-images.githubusercontent.com/213565/101675855-34d18980-3a0f-11eb-95c1-ca352158d9d4.png)

### Additional Considerations
- It does not look like this widget is currently available in other languages, so I only wrote the errors in English. Is there i18n work needed in this PR?
- This change mostly benefits users navigating the page visually. For screen readers, I added `aria-invalid` to form fields when they have errors. It’s not as helpful as the full error messages, but it is better than no feedback.

### Testing
I manually tested the following cases:
- Search without entering anything
- Search with incorrect county name. Change to correct county name and see results.
- Search with incorrect activity name. Change to correct activity name and see results.
- Search correct county and activity. Change to an incorrect activity. See error.

I manually tested my change on Chrome, Safari, Firefox and iOS Chrome/Safari (these were the browsers available to me). I also tried the change using VoiceOver on my Mac.

I did not see a place to put automated tests. Are automated tests needed for this PR?
